src/libcollections/slice.rs

```diff
@@ -343,15 +343,24 @@ impl<T> [T] {
         core_slice::SliceExt::last_mut(self)
     }
 
-    /// Returns the element of a slice at the given index, or `None` if the
-    /// index is out of bounds.
+    /// Returns a reference to an element or subslice depending on the type of
+    /// index.
+    ///
+    /// - If given a position, returns a reference to the element at that
+    ///   position or [`None`] if out of bounds.
+    /// - If given a range, returns the subslice corresponding to that range,
+    ///   or [`None`] if out of bounds.
+    ///
+    /// [`None`]: ../../std/option/enum.Option.html#variant.None
```

Comment:
There is an extra "..". Should be: `../std/option/enum.Option.html#variant.None`.